akvo/rsr/models/project.py

```diff
@@ -554,6 +554,19 @@ def reporting_partner(self):
                 iati_organisation_role=Partnership.IATI_REPORTING_ORGANISATION)
         except ObjectDoesNotExist:
             return None
+        except MultipleObjectsReturned:
+            # A project with multiple reporting organisations should not happen, but in practice
+            # it sometimes does unfortunately. In these cases we check if there's one "primary
+            # reporter" and return that. If not, we return the first reporting organisation.
+            primary_reporters = self.partnerships.filter(
+                iati_organisation_role=Partnership.IATI_REPORTING_ORGANISATION,
+                secondary_reporter='0'
```

Comment:
Haven't tested, but I'm guessing this filter is more correct:
```python
.filter(iati_organisation_role=Partnership.IATI_REPORTING_ORGANISATION).exclude(is_secondary_report=True)
```
since it's a NullBooleanField.